starknet_py/abi/v1/abi.ebnf

```diff
@@ -0,0 +1,21 @@
+IDENTIFIER: /[a-zA-Z_][a-zA-Z_0-9]*/
+
+start: type
+     | struct
+
+type: "core" "::" core_value
+    | type_unit
+
+core_value: "felt" "252"                                        -> type_felt
+          | "integer" "::" "u" INT                              -> type_uint
```

Comment:
Shouldn't this somehow distinguish between uints like 128, 256 etc?

Reply:
I think the type of uint will definitely influence the calldata we are sending so distinguishing between them will be necessary.